### ISSUE TITLE
Custom Message along with the notifications core message

### ISSUE THREAD

**Issue by saikatharryc** [YOU54F/cypress-slack-reporter#621]
@YOU54F  I think it will also be helpful to certain users, who can actually set some custom static message, which can be passed through the CLI itself?

do you have something like that? and let me know how this feature request sounds like? I can make a PR for that, incase makes sense.

**Comment by YOU54F** [comment]
No don't have anything in place for that at the moment.

There are a few different items you can attach to the message, ie snippets etc, but you may just want some additional plain text. 

I would definitely look at merging any proposed PR so if you do feel like it, feel free! :)

**Comment by saikatharryc** [comment]
alright! and thank you. I could not get much time on this, but I feel it will be a great addition.

**Comment by YOU54F** [comment]
Heye @saikatharryc 

Would you want to replace any of the text, or just add something additional? If so, where would you add it?

https://github.com/YOU54F/cypress-slack-reporter/blob/master/samples/alerts.png

I'm looking to refactor this soon, so will support this as a feature

**Comment by saikatharryc** [comment]
this all is fine, for me somehow it does not show the branch name.
so , if can pass some additional text dynamically would be great. as header. or initial text. which will/might override `Cypress test run failed`
<img width="357" alt="Screen Shot 2020-07-19 at 1 55 51 PM" src="https://user-images.githubusercontent.com/15653282/87870631-909b6780-c9c7-11ea-8dac-bdc48864080f.png">

**Comment by YOU54F** [comment]
@saikatharryc what command are you using to run it and which CI server? I can see you don't have any git info in your message, which would be why the branch isn't showing.

I have a branch I am working on and you will be able to pass `--custom-text "your text here"` and have it show up 👍🏽 Will let you know when it is out

**Comment by saikatharryc** [comment]
> I have a branch I am working on and you will be able to pass --custom-text "your text here" and have it show up 👍🏽 Will let you know when it is out

That's awesome!!!!
Thanks a lot for this,

and , 
I'm using this command:
"cypress-slack-reporter": "^0.7.1"
```
npx cypress-slack-reporter --report-dir ./mochareports --vcs-provider=none --ci-provider=custom --custom-url=<MY_CUSTOM_GITHUB_ENTERPRISE_URL>
```

**Comment by YOU54F** [comment]
This is now available on the latest version `1.0.2`

```
	npx cypress-slack-reporter --custom-text "some custom text"
```

Ahh yeah with vcs-provider set to none, you wont get any git metadata
